templates/docker-compose.template

```diff
@@ -0,0 +1,101 @@
+version: '2'
+
+services:
+
+  gateway-config:
+    image: nebraltd/hm-config:{{CONFIG_VERSION}}
+    depends_on:
+      - dbus-session
+      - diagnostics
+    environment:
+      - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
+      - DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket
+      - DBUS_SESSION_BUS_ADDRESS=unix:path=/session/dbus/session_bus_socket
+    privileged: true
+    network_mode: host
+    cap_add:
+      - NET_ADMIN
+    volumes:
+      - miner-storage:/var/data
+      - dbus:/session/dbus
+    labels:
+      io.balena.features.sysfs: 1
+      io.balena.features.kernel-modules: 1
+      io.balena.features.dbus: 1
+    stop_signal: SIGINT
+
+  packet-forwarder:
+    image: nebraltd/hm-pktfwd:{{PKTFWD_VERSION}}
+    depends_on:
+      - helium-miner
+    restart: always
+    privileged: true
+    volumes:
+      - pktfwdr:/var/pktfwd
+    environment:
+      - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
+
+  helium-miner:
+    image: nebraltd/hm-miner:{{ARCH}}-{{MINER_VERSION}}
+    depends_on:
+      - dbus-session
+      - diagnostics
+    expose:
+      - "1680"
+      - "4467"
+    ports:
+      - "44158:44158/tcp"
+    volumes:
+      - miner-storage:/var/data
+      - miner-log:/var/log/miner
+      - pktfwdr:/var/pktfwd
+      - dbus:/session/dbus
+    cap_add:
+      - SYS_RAWIO
+    devices:
+      - {{I2C_DEVICE}}:{{I2C_DEVICE}}
+    restart: on-failure
+    environment:
+      - DBUS_SYSTEM_BUS_ADDRESS=unix:path=/session/dbus/session_bus_socket
+      - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
+
+  diagnostics:
+    image: nebraltd/hm-diag:{{DIAGNOSTICS_VERSION}}
+    environment:
+      - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
+      - DIAGNOSTICS_VERSION={{DIAGNOSTICS_VERSION}}
+      - DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket
+    volumes:
+      - pktfwdr:/var/pktfwd
+      - miner-storage:/var/data
+    ports:
+      - "80:7000"
```

Comment:
Did we decide to change this to 7000 in the end instead of 5000?

Might have knock on effects to HPT?

Reply:
We can keep using 5000 or 7000. From HPT side all this is exposed via docker on the miner at port 80. So HPT does not have or need direct access to ports within the containers.